components/pagination/Pagination.js

```diff
@@ -0,0 +1,130 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import { LinkButton } from '../button';
+import { TextSmall } from '../typography';
+import { IconChevronLeftMediumOutline, IconChevronRightMediumOutline } from '@teamleader/ui-icons';
+import cx from 'classnames';
+import omit from 'lodash.omit';
+import theme from './theme.css';
+
+class Pagination extends PureComponent {
+  static propTypes = {
+    inverse: PropTypes.bool,
+    nextPageText: PropTypes.string,
+    numItems: PropTypes.number.isRequired,
+    numItemsPerPage: PropTypes.number,
+    maxNumPagesVisible: PropTypes.number,
+    currentPage: PropTypes.number.isRequired,
+    onPageChange: PropTypes.func.isRequired,
+    prevPageText: PropTypes.string,
+    className: PropTypes.string,
+  };
+
+  static defaultProps = {
+    currentPage: 1,
+    inverse: false,
+    numItemsPerPage: 10,
+    maxNumPagesVisible: 7,
+  };
+
+  constructor() {
+    super(...arguments);
+
+    this.handleChange = ::this.handleChange;
+  }
+
+  getNumPages() {
+    return Math.ceil(this.props.numItems / this.props.numItemsPerPage);
+  }
+
+  handleChange(page) {
+    this.props.onPageChange(page);
+  }
+
+  render() {
+    const { className, currentPage, inverse, maxNumPagesVisible, nextPageText, prevPageText, ...others } = this.props;
+    const numPages = this.getNumPages();
```

Comment:
PP: A bit overkill to have a function doing this? If you destructure `numItems` and `numItemsPerPage` you could just do `const numPages = Math.ceil(numItems / numItemsPerPage);`. I think it's too much because you're not going to use this function more than once in this component. But I get why you would split it up though :)

Reply:
Good idea, and so I can drop the `omit` part!

 :ok: by @driesd on 20/12/2017 08:50:52